integration.json

```diff
@@ -0,0 +1,6 @@
+{
+  "bundle-install": "github.com/paketo-community/bundle-install",
+  "bundler": "github.com/paketo-community/bundler",
+  "mri": "github.com/paketo-community/mri",
+  "builder": "index.docker.io/paketobuildpacks/builder:full-cf"
```

Comment:
Looks like the buildpack won't run on `base` because it wants there to be `curl` installed on the `$PATH`. Let's include `curl` as a direct dependency that is installed by this buildpack so that we can run it on `base`.